jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/complex/Modal.java

```diff
@@ -0,0 +1,13 @@
+package com.epam.jdi.light.ui.bootstrap.elements.complex;
```

Comment:
Move to 
com.epam.jdi.light.ui.bootstrap.elements.composite

Reply:
For now we have .epam.jdi.light.ui.bootstrap.elements.common and .epam.jdi.light.ui.bootstrap.elements.complex. Should I create new one? complex in jdi-light-bootstrap? (now we have composite in jdi-light-bootstrap-tests only)

Reply:
Yes, you should create new package in jdi-light-bootstrap:
com.epam.jdi.light.ui.bootstrap.elements.composite